addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/TimestampDisclosureScanRule.java

```diff
@@ -188,6 +195,18 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         }
     }
 
+    private boolean isFontRequest(HttpMessage msg) {
+        try {
+            final String path = msg.getRequestHeader().getURI().getPath();
```

Comment:
We only add final when really needed.

Reply:
I think that was due to a copy/paste from some core content. Will remove.